code/game/objects/items/weapons/surgery_tools.dm

```diff
@@ -262,3 +262,84 @@
 	desc = "A favorite among skeletons. It even sounds like a skeleton too."
 	icon_state = "bone-mender"
 	surgery_speed = 0.5
+
+/obj/item/weapon/organfixer
+	name = "organ fixer"
+	desc = "A device used to fix internal organs."
+	icon = 'icons/obj/surgery.dmi'
+	item_state = "scientology" // TODO: Draw a proper handheld sprite; For now it looks fine ~Toby
```

Comment:
![yey](https://user-images.githubusercontent.com/48761132/82868226-ee19b780-9f34-11ea-929a-ccfaa5637b7a.png)
better fix it because i can see it through my ass

Reply:
Когда-нибудь, когда найдётся добрый спрайтер. Ну или могу вообще убрать item_state, как у большинства инструментов.